experimental/lando-acquia/lib/utils.js

```diff
@@ -24,6 +24,20 @@ exports.writeAcliUuid = uuid => {
   return false;
 };
 
+exports.writeEnvUuids = data => {
+  const file = './acquia-envs.json';
```

Comment:
@mikemilano here i'm writing the env uuids for later use into `/app/acquia-envs.json`, but i'd rather put that in `/var/www/.acquia/acquia-envs.json` or somewhere out of the way, but I had trouble doing that.